src/sentry/web/frontend/react_page.py

```diff
@@ -29,6 +31,9 @@ def handle_react(self, request):
 # generic
 class ReactPageView(OrganizationView, ReactMixin):
     def handle(self, request, **kwargs):
+        if 'project_id' in kwargs and request.GET.get('onboarding'):
+            project = Project.objects.filter(slug=kwargs['project_id']).first()
```

Comment:
I had this initially, but I think it requires you pass "project_slug" instead of project_id, which is why I had originally changed the web/urls.py route to take project_slug, instead of project_id.